Tests/NimbleTests/Matchers/BeGreaterThanOrEqualToTest.swift

```diff
@@ -41,6 +41,10 @@ final class BeGreaterThanOrEqualToTest: XCTestCase, XCTestCaseProvider {
         expect(1) >= 0
         expect(NSNumber(value:1)) >= 1
         expect(NSNumber(value:1)) >= NSNumber(value:1)
+        expect(2.5) >= 2.5
```

Comment:
Since I do not use `swift`, I did not know whether float and double are supported.
So, I added some tests.